vuepress/docs/next/tutorials/create/mfe/widget-configuration.md

```diff
@@ -98,7 +98,7 @@ npm start
 npm run build
 ```
 
-6. Load the updated `my-widget` files into Entando as was done for the [React MFE tutorial](./react.md#upload-the-react-files)
+6. Load the updated `my-widget` files into Entando as was done for the [React MFE tutorial](../../../../v7.0/tutorials/create/mfe/react.md#upload-the-react-files)
```

Comment:
This should be a relative link so it points to the current version, rather than directly to v7.0

Reply:
@nshaw this is a tricky one because this section no longer exists in the current version, and it's not just a matter of renaming - we killed the content. i didn't look at the affected page closely enough to get a sense of how we might also modify it for v5, if at all, so just pointed back to 7.0 instead of removing the link reference and listing all the steps explicitly.

Reply:
@Lyd1aCla1r3 ah, understood. I have some ideas for it based on the Tableau widget conversion I'm working on so we can discuss on our next call.